app/admin/applicant_details.rb

```diff
@@ -78,7 +79,11 @@
     end
     column :us_citizen
     column :demographic do |d| 
+      if d.demographic_name == "Other" && d.demographic_other.present?
+        "#{d.demographic_name} - #{d.demographic_other}"
+      else
         d.demographic_name
+      end
```

Comment:
The code in columns 81 - 86 is repeated three times.
If you replace demographic_name method in the Demographic model with 
```
  def demographic_name
    if self.demographic_other.present?
      Demographic.find(self.demographic).name + " - " + self.demographic_other
    else
      Demographic.find(self.demographic).name
    end
  end
```
in the app/admin/applicant_details.rb you can do
```
    column :demographic do |d| 
      d.demographic_name
    end
```

And line 155 will work correctly too.

Reply:
I updated methid names and made them more Rails idiomatic